build-cassandra-bin.bash

```diff
@@ -24,7 +24,7 @@ fi
 # VERSION SETTINGS
 CASSANDRA_VERSION="3.0.10"
 METRICS_INTERFACE_VERSION="3" # Cassandra 2.2+ uses metrics3, while <= 2.1 uses metrics2.
-STATSD_REPORTER_VERSION="4.1.2"
+STATSD_REPORTER_VERSION="4.1.2-SNAPSHOT"
```

Comment:
Could include a comment that the custom version is needed due to usage of the 3.1.0 interface in Cassandra 3.x(?)